bin/compile.js

```diff
@@ -106,7 +107,7 @@ async function writeIndex(targetDir, filenames) {
 async function build() {
   let userConfig = {}
   try {
-    const pathPrefix = isWindows() ? 'file://' : ''
+    const pathPrefix = process.platform === 'win32' ? 'file://' : ''
```

Comment:
Not sure why you switch back to this formula.
Does the `isWindows()` function not work as expected?

Reply:
Nah. I'm reverting the commit. I pulled the last package from NPM and that's just what its code currently has. =) 

I didn't realize every commit I do would email you! Whoops!